UnrealTournament/SystemLocalized/itt/Setup.itt

```diff
@@ -218,4 +218,4 @@ IDC_CompleteReboot="Installazione completata!\Prima di giocare, è necessario ri
 
 [IDDIALOG_WizardDialog]
 ; EN: IDC_WizardDialog="Unreal Tournament Setup"
-IDC_WizardDialog="Archivio Unreal Tournament"
+IDC_WizardDialog="Installazione di Unreal Tournament"
```

Comment:
Remember to also check Setup.itt and SetupUnrealPatch.itt in Unreal and sync your changes there too.